### ISSUE TITLE
Default to track language in mentor comments

### ISSUE THREAD

**Issue by SaschaMann** [exercism/exercism#5306]
It would be convenient if triple backtick code blocks in mentor comments would default to the track's language to get syntax highlighting without having to type the language manually.

**Comment by kotp** [comment]
That would be irritating to me.  I am very much in the habit of "driving".  I mean specifying a language, and without a language, I want it to be just mono-space text.

**Comment by SaschaMann** [comment]
Could be a user-option I guess

**Comment by ErikSchierboom** [comment]
Hello 👋

With the launch of Exercism v3, we are **closing all issues in this repository** to help give us a clean slate to detect new problems. If this issue is still relevant to Exercism v3 (e.g. it's a feature that we haven't implemented in v3, or a bug that still exists), please reopen it and we will review it and post an update on it as soon as we get chance.

Thanks for helping make Exercism better, and we hope you enjoy v3 🙂

**Comment by SaschaMann** [comment]
AFAICT this is still relevant and should be reopened.

**Comment by joshgoebel** [comment]
I agree with @kotp here.  I often use "untagged" text boxes for bad code (don't do this example) and then use syntax highlighting for the GOOD code... or if I want to insert pseudo-code (plaintext, etc).  I think I'd prefer to see a keyword macro that inserted a 

````
```current-lang
[cursor]
```
````

block at the current position for people who wanted a fast way to do this.

**Comment by kotp** [comment]
I do agree that a user opt-in would be nice to have, or perhaps there is an icon added to do exactly this: `<current_lang>` or whatever single character/emoji thing represents the current language, perhaps even our Exercism logo icons could go there.  But retain the "no language" markdown button.  But typing 

~~~
```
~~~

probably should not insert the current language for us.

---

Also, yeah, I think this should be reopened and discussed further.
